Fix footer layout

shaina33 opened the pull request:
# Description of changes
Currently, the layout of the footer links makes them look haphazard, and sometimes unusable, so I'm working to correct that. The links section of the footer jsx contains 2 blockGroups of 5 links each, so I kept these and styled them into 2 columns, mainly by adjusting flex and padding properties. I also changed the 2 outerFooterGroups into 1 outerFooterGroupSocial and 1 outerFooterGroupLinks so that they could be styled separately.

With my changes, the links look good at screen widths <480px and >810px. I'm still working on the in-between range. At these widths, some links wrap to 2 lines, disrupting the alignment and cutting off the bottom link. I don't want to make the text much smaller, so I'm thinking it may be best to adjust the media queries so that the footer elements stack vertically below about 810px, instead of below 480px. Would that be alright, @kylemh? That would also resolve the issue of the email address being cut-off at intermediate screen widths. To reduce blank space, the 4 social buttons could be kept on 1 line.

Per your suggestion, I added some more padding between the links, but I didn't do the full 1rem because it seemed like too much. Let me know if you think it needs more. Thanks for your help!

# Issue Resolved
<!-- Keeping the format 'Fixes #123' will automatically close the issue when this PR is merged -->
Fixes #816 

# Screenshots
1300 px:
![screen shot 2018-01-23 at 5 13 37 pm](https://user-images.githubusercontent.com/17550347/35303789-306e926a-0061-11e8-9d62-cacd2ba81598.png)
650px:
![screen shot 2018-01-23 at 5 14 47 pm](https://user-images.githubusercontent.com/17550347/35303803-40511518-0061-11e8-8aee-9d308670b85e.png)
470px:
![screen shot 2018-01-23 at 5 15 06 pm](https://user-images.githubusercontent.com/17550347/35303815-4c5ec382-0061-11e8-917b-00f4fe66ea43.png)

kylemh commented:
One thing that may help is gaining control of the font sizes.

It's my fault for letting it through, but please copy-paste this style-sheet into `src/scenes/home/branding/logos/logos.css` - you'll then be able to actually apply font-size changes and see them work.

First add `className={styles.logosInfo}` to the `<p>` element on line 39 of `logos.js` and then...

```
.marginSpace {
  margin: 5em 0;
}

.sectionPadding {
  margin: 2em 0 0;
  padding: 3em 0 0;
  border-top: 1px solid #ddd;
  border-bottom: 1px solid #ddd;
}

.sectionPadding:last-child {
  border: none;
}

.logosInfo {
  width: 60%;
  font-size: 1.2rem;
  text-align: justify;
}

.logosInfo a {
  color: #249cbc;
  font-size: 1.3rem;
  text-decoration: none;
}

.logoImages {
  width: 100%;
  text-align: center;
}

.smallLogos {
  display: flex;
  align-items: center;
  flex-flow: row;
  padding: 1rem 0;
}

.stackLogos {
  width: 50%;
}

.starLogos,
.stackedLogos {
  width: 33.33%;
}

.smallMedalLogos {
  text-align: center;
}

.medalLogos {
  margin: 1em;
  display: inline-block;
}

.medalLogos img {
  max-width: 130px;
}

.largeLogos {
  display: flex;
  align-items: center;
  flex-flow: column;
}

.largeLogos .stackLogos,
.largeLogos .starLogos {
  width: 625px;
}

.largeLogos .stackedLogos {
  width: 460px;
}

.largeLogos img {
  width: 100%;
  max-width: 100%;
}

@media (max-width: 360px) {
  .smallLogos .stackLogos,
  .smallLogos .starLogos,
  .smallLogos .stackedLogos {
    width: 100%;
    max-width: 200px;
  }
}

@media (max-width: 424px) {
  .logosInfo {
    width: 100%;
  }
  .smallLogos {
    flex-flow: column;
  }
  .smallLogos .stackLogos,
  .smallLogos .starLogos,
  .smallLogos .stackedLogos {
    width: 100%;
    max-width: 320px;
  }
  .smallLogos img {
    width: 100%;
    max-width: 100%;
  }
  .largeLogos .stackLogos,
  .largeLogos .starLogos {
    width: 100%;
  }
  .largeLogos .stackedLogos {
    width: 100%;
  }
}

@media (min-width: 425px) and (max-width: 768px) {
  .logosInfo {
    width: 80%;
  }
  .smallLogos img {
    width: 90%;
    max-width: 100%;
  }
}

@media (min-width: 769px) and (max-width: 1024px) {
  .logosInfo {
    width: 70%;
  }
  .smallLogos .starLogos img,
  .smallLogos .stackedLogos img {
    width: 90%;
  }
}
```

kylemh commented:
I'll give a better review tomorrow night

shaina33 commented:
I think it's getting close! I changed the vertical breakpoint to 768px and adjusted spacing so it all lays nicely. I added 0.75rem of vertical margin to the links. And it seems to be looking good on iOS Safari too, here's a screenshot from my phone.

![screen shot 2018-01-23 at 5 15 06 pm](https://user-images.githubusercontent.com/17550347/35362021-a56dc0a6-0131-11e8-888a-fae26cd16362.png)

kylemh commented:
I added some minor changes so I don't have to keep annoying you with pixel perfect BS.

I love it so much - you're amazing! Great first PR.

shaina33 commented:
Thank you! I'm looking forward to making more contributions!
